docs/monitoring.md

```diff
@@ -479,11 +479,14 @@ can be identified by their `[attempt-id]`. In the API listed below, when running
     <td><code>/applications/[app-id]/stages/[stage-id]</code></td>
     <td>
       A list of all attempts for the given stage.
+        <br><code>?details=true</code> lists all attempts with task data for the given stage.
+        <br><code>?taskStatus=[RUNNING|SUCCESS|FAILED|KILLED|PENDING]</code> lists only those tasks with the specified task status. Query parameter taskStatus takes effect only when <code>details=true</code>.
```

Comment:
It would be great to document that this also supports multiple `taskStatus`s, such as `details=true&taskStatus=success&taskStatus=failed`.

Reply:
yea